src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -539,8 +544,8 @@ double G1Policy::predict_retained_regions_evac_time() const {
     num_regions++;
   }
 
-  log_trace(gc, ergo, heap)("Selected %u of %u retained candidates taking %1.3fms additional time",
-                            num_regions, list.length(), result);
+  log_trace(gc, ergo, heap)("Selected %u of %u retained candidates (unreclaimable %u) taking %1.3fms additional time",
```

Comment:
I actually think calling it "pinned", instead of "unreclaimable", is more informative (to users/dev). (And other places when it is shown in logs.)